docs/setting-up-local-development.md

```diff
@@ -76,6 +76,10 @@ These instructions assume you're using a package manager for your OS:
   $ sudo apt-get install rbenv ruby-build
   $ rbenv install X.Y.Z
   ```
+  If you are having trouble getting the version required, clone ruby-build via git instead of apt-get:
```

Comment:
I am actually planning to update this one, because I removed that when I made the instructions for Windows Subsystem for Linux 
In that case though, what happened is that ruby-build did not have version 2.3.3 (the one provided by apt-get). Cloning from github solved that.